newsletter-bundle/src/Resources/contao/models/NewsletterChannelModel.php

```diff
@@ -22,6 +22,7 @@
  * @property string  $template
  * @property string  $sender
  * @property string  $senderName
+ * @property string  $mailer_transport
```

Comment:
Please also adjust the `find*()` and `count*()` methods.

Reply:
Done.